packages/pathway-mapper/src/managers/GenomicDataOverlayManager.ts

```diff
@@ -453,7 +453,7 @@ export default class GenomicDataOverlayManager {
       }
 
       let colorString = "";
-      if (percent) {
+      if (percent && Number(percent.substr(1)) !== 101) {
```

Comment:
Relying on a very specific implementation detail where we assume that the only invalid number is `-101` may not be the best solution. We should instead check if the numerical value falls within the valid interval of `[0, 100]`.
```suggestion
      if (percent &&  Math.abs(Number(percent)) >= 0 && Math.abs(Number(percent)) <=100) {
```